app/javascript/packages/components/full-screen.tsx

```diff
@@ -108,20 +108,14 @@ function FullScreen(
       className={`full-screen bg-${bgColor}`}
     >
       {children}
-      {!hideCloseButton && (
-        <button
-          type="button"
-          aria-label={t('account.navigation.close')}
-          onClick={onRequestClose}
-          className="full-screen__close-button usa-button padding-2 margin-2"
-        >
-          <img
-            alt=""
-            src={getAssetPath('close-white-alt.svg')}
-            className="full-screen__close-icon"
-          />
-        </button>
-      )}
+      <button
+        type="button"
+        aria-label={t('account.navigation.close')}
+        onClick={onRequestClose}
+        className={hideCloseButton ? '' : 'full-screen__close-button usa-button padding-2 margin-2'}
```

Comment:
I see the comment above about z-index effectively hiding this button. Does that apply for screen readers as well? My concern is that someone using a screen reader could still find this button by navigating content, but the way we're implementing it seems to imply we don't want it to be available on the page.

I guess I'm not really following why we're making this change at all.

Reply:
Ah, thanks. Here's the "why" for this change:
- This is currently the only place in the codebase where `hideCloseButton` is in use.
- We don't want to show the close button (gray with an X icon) because it overlaps the "close" text in the top right corner and looks bad on the selfie capture screen.
- If I try to use `hideCloseButton` as it's currently implemented in `main`, it does not work for selfie capture (see details below) because the Acuant "close" functionality appears after the focus trap is activated, which makes the focus trap error out.
- This is my (inelegant) way of solving the problem of having two overlapping close functionalities and hiding the close button with the "X".

Other thoughts:
- Thinking about this I realized that the hidden close button should continue to work, so I did [this](https://github.com/18F/identity-idp/pull/10228/commits/a9f05755f6d0de46f6b7c736c364bcdcd7613743).
- I'm open to other solutions, I couldn't come up with anything else that fit within a reasonable scope.

Details on why `hideCloseButton` doesn't work:
- When I get to the front/back/selfie screen all is well.
- But clicking on the selfie capture box does not open the selfie capture as expected.
- Instead of opening selfie capture I get a console log from the focus-trap library that says something to the effect of "The focus trap needs to have a clickable element at all times".
- I think that this is happening because there's some amount of time where the focus trap is active on `acuant-face-capture-container` but Acuant hasn't rendered in the `close` text yet.

Reply:
Yeah that looks a lot better 👍

Reply:
🚀 Thanks for all the comments! I really appreciate it.